src/app/containers/App/App.jsx

```diff
@@ -82,7 +82,7 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
         });
       });
 
-      route.getInitialData(location.pathname).then(data => {
+      route.getInitialData(location.pathname, nextService).then(data => {
```

Comment:
Or just check if we've passed service in the function, and let the useData hook handle client side data fetching